lib/checks/expired_whitelist_entries.rb

```diff
@@ -0,0 +1,20 @@
+module Checks
+  class ExpiredWhitelistEntries
+
+    def initialize(checker_db, whitelist)
+      @checker_db = checker_db
+      @whitelist = whitelist
+    end
+
+    # find whitelist entries which have expired
+
+    def run_check
+
```

Comment:
can you be more specific? Do we have particular linting rules you are manually applying? If so I suggest we wait until lint and rubocop are added to this repo (the task is already on our list).